src/components/bottom/BottomPanel.svelte

```diff
@@ -27,6 +27,11 @@
         <!-- The live text and info box -->
         <LiveGraphInformationSection />
         <ConnectedLiveGraphButtons />
+        {#if false}
```

Comment:
This is false on purpose. We need the string to be referenced for the tests to pass.